demos/scatter-plot.html

```diff
@@ -3,15 +3,44 @@
         <div class="row">
             <div class="col-md-8">
                 <article>
-                    <h2>Scatter Plot</h2>
-                    <div class="js-simple-scatter-plot-container scatter-plot-container card--chart"></div>
+                    <h2>Scatter Plot with single source and vertical grid</h2>
+                    <div class="js-scatter-plot-with-single-source scatter-plot-container card--chart"></div>
                 </article>
             </div>
             <div class="col-md-4 sidebar">
                 <h3>The code</h3>
                 <pre><code class="language-javascript">
-scatterPlot
+scatter
     .width(750)
+    .grid('vertical');
```

Comment:
The screenshot seems to show a horizontal grid instead of a vertical one, is that correct?

Reply:
Yes! Made changes multiple times, will change